ci-operator/step-registry/upi/conf/gcp/upi-conf-gcp-commands.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+### TODO: Move this to Dockerfile.upi.ci
+export HOME=/tmp
+pip install --user pyyaml
+
+pushd ${SHARED_DIR}
+
+### Empty the compute pool (optional)
+python -c '
+import yaml;
+path = "install-config.yaml";
+data = yaml.full_load(open(path));
+data["compute"][0]["replicas"] = 0;
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+### Create manifests
+cp ${SHARED_DIR}/install-config.yaml ${SHARED_DIR}/install-config.yaml.bak
+openshift-install --dir="${SHARED_DIR}" create manifests 2>&1 | grep --line-buffered -v password &
```

Comment:
https://github.com/openshift/release/blob/a26c06b316a6edae94861b463af645ff0a7dee41/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L29-L31

https://github.com/openshift/release/blob/a26c06b316a6edae94861b463af645ff0a7dee41/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L48

i think we shouldn't use `shared_dir` but a different directory do that files in `shared_dir` are by specific choice.

Reply:
done